containers/ddev-webserver/ddev-webserver-etc-skel/etc/skel/.bashrc

```diff
@@ -116,6 +116,9 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && source "$NVM_DIR/nvm.sh"
 [ -s "$NVM_DIR/bash_completion" ] && source "$NVM_DIR/bash_completion"
 
+# bash completion for npm
+[ -n "$(type -t npm)" ] && eval "$(npm completion)"
+
```

Comment:
Is this the right file to put this in for it to apply to the webserver? I haven't touched anything to do with the containers before and the naming here is a little confusing.

Reply:
This seems like it ought to be a fine place, of course only for bash, and only works if login shell.

Reply:
It's fine that it's only for bash, because this is inside the web container and not on the host.

The auto completion script is explicitly executed in the container in bash, and the result gets passed back to cobra which then outputs it correctly for whichever she'll the host is using.

Reply:
People *can* use zsh inside the container, but it's pretty exotic, and not sure it affects this case anyway. https://ddev.com/blog/oh-my-zsh-using-custom-commands-and-other-goodies-to-add-to-ddev/